Sources/TuistGraph/Models/IDETemplateMacros.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+
+public struct IDETemplateMacros: Codable, Hashable {
+    enum CodingKeys: String, CodingKey {
+        case fileHeader = "FILEHEADER"
+    }
+
+    public let fileHeader: String?
+
+    public init(fileHeader: String?) {
+        // Xcode by default adds extra newline at the end, so if it is present, just remove it
```

Comment:
minor suggestion: (not sure where is best to add this logic)

Perhaps automaitcally add a leading space?

```swift
let config = Config(generationOptions: [
    .fileHeaderTemplate("FooBar Inc")
])
```

![Screenshot 2021-03-01 at 08 45 40](https://user-images.githubusercontent.com/11914919/109472917-a21f1480-7a6a-11eb-82b9-9692f1eaf5c0.png)

Reply:
Yeah the implicit comment is a bit annoying, I am thinking about checking for a leading space and adding it if not present and for leading `//` and removing it if present as it can be quite messy

Reply:
Done ✅